Create a global CONTRIBUTING page for repositories which do not override it (#24)
jenkinsci/.github: CONTRIBUTING.md

# Contributing to Jenkins

In the Jenkins project we appreciate any kind of contributions: code, documentation, design, etc.
Any contribution counts, and the size does not matter!
Check out [this page](https://jenkins.io/participate/) for more information and links!

Many plugins and components also define their own contributing guidelines and communication channels. 
There is also a big number of [mailing lists](https://jenkins.io/mailing-lists/) and [chats](https://jenkins.io/chat/).

> | WARNING: This is a default CONTRIBUTING page for all repositories in Jenkins. Every plugin/component is its own sub-project which may deviate in its rules. Guidelines and channel links in repository CONTRIBUTING/README pages, if any, take precedence over this page. |

Reviewer: The text is now a lot better, but presentation (bold, all-caps "WARNING") still looks scary -- like something that not reading would have _very_ negative consequences.

If this is intentional, that's fine. I don't think this warrants it, though.

Author: ```suggestion
| NOTE: This is a default CONTRIBUTING page for all repositories in Jenkins. Every plugin/component is its own sub-project which may deviate in its rules. Guidelines and channel links in repository CONTRIBUTING/README pages, if any, take precedence over this page. |
```

# Contributing to Jenkins

> [![Contributing](https://img.shields.io/badge/contributing-guidelines-blue)](https://jenkins.io/participate/)

Reviewer: But why?

Author: hm?

Reviewer: I don't understand the use of badges as, apparently, decorative elements.

Author: It was rather a quick link for those who do not want to read text. Happy to revert it

```suggestion
```

Reviewer: Sure, having a link makes sense, I'm just confused about the badge as link label. Why not write something like the following instead?

> Want to learn more about the various ways of contributing to Jenkins? See our guides on jenkins.io.

Seems more accessible that way (less ambiguous).

In the Jenkins project we appreciate any kind of contributions: code, documentation, design, etc.
Any contribution counts, and the size does not matter!
Check out [this page](https://jenkins.io/participate/) for more information and links!

Many plugins and components also define their own contributing guidelines and communication channels. 
There is also a big number of [mailing lists](https://jenkins.io/mailing-lists/) and [chats](https://jenkins.io/chat/).

> | WARNING: This is a default CONTRIBUTING page for all repositories in Jenkins. It might be overridden in repositories, and contributing guidelines maybe also defined in other places README. |

Reviewer: Very important, but perhaps less scary? Explain that in a way, every plugin is its own subproject, and its maintainers may deviate in their rules, so if the repo has a README etc., it takes precedence over this?

Author: ```suggestion
| WARNING: This is a default CONTRIBUTING page for all repositories in Jenkins. Every plugin/component is its own sub-project which may deviate in its rules. Guidelines and channel links in repository CONTRIBUTING/README pages, if any, take precedence over this page. |
```

Author: @daniel-beck WDYT?
